src/lib/rmt/wizard_ssl_page.rb

```diff
@@ -117,7 +137,17 @@ def remove_alt_name_handler
 
   def run
     if @cert_generator.server_cert_present?
-      Yast::Popup.Message(_('SSL certificates already present, skipping generation.'))
+      if @cert_generator.ca_encrypted?
+        current_ca_password = RMT::SSL::CurrentCaPasswordDialog.new.run
+
+        if !current_ca_password
+          Report.Error(_('SSL certificate password not provided, skipping import.'))
+        elsif @cert_generator.valid_password?(current_ca_password)
+          Yast::Popup.Message(_('SSL certificates already present and password is valid, skipping generation.'))
+        end
+      else
+        Yast::Popup.Message(_('SSL certificates already present, skipping generation. Please consider encrypting your CA private key!'))
```

Comment:
That text is quite long, in some translations it might be even longer. To avoid clipping the text in the 80x25 text mode add a `\n` line break in the middle.